lib/SQLiteReporter.cpp

```diff
@@ -215,6 +206,24 @@ void Mutang::SQLiteReporter::reportResults(const std::unique_ptr<Result> &result
   outs() << "Results can be found at '" << databasePath << "'\n";
 }
 
+#pragma mark -
+
+std::string SQLiteReporter::getCallerPathAsString(std::vector<std::string> callerPath) {
```

Comment:
That's alright, that's one of the benefits of code reviews. :-) To answer your question, both are equivalent semantically. In my experience the second type is most commonly used, so I'd recommend that one.